lib/common/src/Dom/Document.php

```diff
@@ -1358,16 +1370,22 @@ private function restoreAmpEmojiAttribute($html)
      *
      * ⚠️ This is not cryptographically secure!
      *
-     * @param int $min Lower limit for the generated number
-     * @param int $max Upper limit for the generated number
+     * @param int $min Lower limit for the generated number. Defaults to 0.
+     * @param int $max Upper limit for the generated number. Defaults to max random number.
      * @return int A random number between min and max
      */
-    private function rand($min = 0, $max = 0)
+    private function rand($min = 0, $max = null)
```

Comment:
Actually we might as well just remove the arguments altogether since we never use them. Then the default max will be used.

Reply:
Done in bfea8ec45